<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoiseReductionOptionsWidgetClass</class>
 <widget class="QWidget" name="NoiseReductionOptionsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Noise Reduction Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="m_groupBox_temporalFiltering">
     <property name="title">
      <string>Temporal filtering</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="m_groupBox_sphara">
     <property name="title">
      <string>SPHARA</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="m_checkBox_activateSphara">
        <property name="text">
         <string>Activate SPHARA</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="m_spinBox_nBaseFctsGrad">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="m_spinBox_nBaseFctsMag">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="m_label_nBaseFctsMag">
        <property name="text">
         <string>Mag:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Number of used base functions (upper limit):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="m_label_nBaseFctsGrad">
        <property name="text">
         <string>Grad:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QComboBox" name="m_comboBox_acquisitionSystem">
        <item>
         <property name="text">
          <string>VectorView</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BabyMEG</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>AcquisitionSystem:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="m_groupBox_ssp">
     <property name="title">
      <string>SSP</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="m_groupBox_compensators">
     <property name="title">
      <string>Compensators</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
